.github/ISSUE_TEMPLATE/bug-report--vs-code-runtime-extension.md

```diff
@@ -0,0 +1,27 @@
+---
+name: "Bug report"
+about: 'Create a report about something that is potential buggy'
+title: ''
+labels: ''
+assignees: ''
+---
+
+>❗ Privacy Alert! The contents copied to your clipboard may contain personal data. Prior to posting to GitHub, please remove any personal data which should not be publicly viewable. https://privacy.microsoft.com/en-US/privacystatement
+
+> The issue text was copied to the clipboard.  Please paste it into this window. Please attach the log file located at the location from the clipboard. Note that this file may contain personal data.
```

Comment:
```suggestion
> If you came here from VS Code, a log file location was copied to the clipboard. Please attach the log file located at the location from the clipboard. Note that this file may contain personal data.
```

Reply:
The log file location is added to the clipboard now, right?

Reply:
Yes